static/src/js/components/SearchPanels/__tests__/SearchPanels.test.jsx

```diff
@@ -154,9 +154,9 @@ describe('SearchPanels component', () => {
 
       expect(collectionResultsPanelProps.activeSort).toBe('')
       expect(collectionResultsPanelProps.sortsArray[0].label).toBe('Relevance')
-      expect(collectionResultsPanelProps.sortsArray[0].isActive).toBe(true)
+      expect(collectionResultsPanelProps.sortsArray[0].isActive).toBe(false)
```

Comment:
The reason this is failing on the remote is because in the `static.config.json` we set the `collectionSearchResultsSortKey` as `-score` we should either keep this using Relevance or otherwise just mock the function call. We should add a few new tests for the new logic we have introduced (the logic for determining the active sort key) or otherwise move that logic to a util and test it there then just ensure that the final response here is correct

Reply:
We should be mocking any config file values used in a test so that this kind of failure doesn't happen in the future.

Reply:
Actually I was wrong so this is actually being passed in the setup method. We had ran into that problem I described about and I correctly thought it was the same issue here. Setup is passing -score so that does not need to be mocked